protobuf-c.spec

```diff
@@ -33,11 +37,17 @@ extensible format. This package provides a code generator and run-time
 libraries to use Protocol Buffers from pure C (not C++).
 %endif
 
-%if 0%{?suse_version} == 0
+# el7 protobuf is too old to build the compiler
+%if "%{?dist}" != ".el7"
```

Comment:
`%if 0%{?rhel} >= 7` is the correct distro test for EL7 (and higher so >= might not be appropriate, but use of `%{rhel}` is).  `%{dist}` should not be used a proxy for distro-flavour testing as a local site policy or derivative distro policy might be to override `%{dist}`.

Reply:
Done